expression/builtin_arithmetic_vec.go

```diff
@@ -403,13 +403,13 @@ func (b *builtinArithmeticMinusIntSig) vecEvalInt(input *chunk.Chunk, result *ch
 		errType = "BIGINT"
 	}
 	for i := 0; i < len(lhi64s); i++ {
-		if result.IsNull(i) {
-			continue
-		}
 		lh, rh := lhi64s[i], rhi64s[i]
 
 		overflow := b.overflowCheck(isLHSUnsigned, isRHSUnsigned, signed, lh, rh)
 		if overflow {
+			if result.IsNull(i) {
```

Comment:
I had tried to ignore check null of the `builtinArithmeticPlusRealSig` expression, following is the result
```
after
BenchmarkVectorizedBuiltinArithmeticFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-VecBuiltinFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-VecBuiltinFunc-8         	  758680	      1525 ns/op
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-NonVecBuiltinFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-NonVecBuiltinFunc-8      	   52083	     21872 ns/op
PASS

before
BenchmarkVectorizedBuiltinArithmeticFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-VecBuiltinFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-VecBuiltinFunc-8         	  560356	      2061 ns/op
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-NonVecBuiltinFunc
BenchmarkVectorizedBuiltinArithmeticFunc/builtinArithmeticPlusRealSig-NonVecBuiltinFunc-8      	   54487	     21818 ns/op
PASS
```
It seems that other expression can be optimized also. Could you help to fix them in this PR ?

Reply:
Done